packages/elements-react/src/locales/en.json

```diff
@@ -260,6 +260,20 @@
   "property.username": "username",
   "property.identifier": "identifier",
   "property.code": "code",
+  "consent.title": "Authorize {party}",
+  "consent.subtitle": "A third party application wants to access information associated with your account {identifier}.",
+  "consent.scope.openid.title": "Public Data",
+  "consent.scope.openid.description": "This application will be able to identify you and read public information about you.",
```

Comment:
Can you please update the descriptions for the different scopes, right now they're some generic text :)